packages/webapp/next.config.js

```diff
@@ -10,8 +10,8 @@ const withBundleAnalyzer = require('@next/bundle-analyzer')({
 
 const securityHeaders = [
```

Comment:
Many changes here, I think it's just formatting, but can we ensure nothing. changed?
It's highly sensitive to make changes in this file if not needed.

Reply:
I tried to revert the changes, but my IDE still changes the formatting. There should not be any changes at all!